### PR TITLE
feat(issue-details-page): Add Successfully Banner - Reprocessing

### PR DESCRIPTION
**Pull request by priscilawebdev**
closes: https://app.asana.com/0/1182975495223897/1199387546700582

### CODE REVIEW

**Review by untitaker — APPROVED** [review]
seems to work for me, but I can't say much about the code

File: src/sentry/static/sentry/app/views/organizationGroupDetails/groupDetails.tsx [hunk 1]
```diff
@@ -165,8 +191,29 @@ class GroupDetails extends React.Component<Props, State> {
...
+      const reprocessingStatus = getGroupReprocessingStatus(data);
+      const {currentTab, baseUrl} = this.getCurrentRouteInfo(data);
+
       if (this.props.params.groupId !== data.id) {
+        if (hasReprocessingV2Feature) {
+          // Redirects to the Activities tab
+          if (
+            reprocessingStatus === ReprocessingStatus.FULLY_REPROCESSED &&
+            currentTab !== TAB.ACTIVITY
+          ) {
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/organizationGroupDetails/groupDetails.tsx:208]: Should these two ifs be merged together?

File: src/sentry/static/sentry/app/components/reprocessedBox.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+              ? tct('All events in this issue were moved during reprocessing. [link]', {
+                  link,
+                })
+              : tct('Events in this issue were successfully reprocessed. [link]', {link})}
+          </span>
+          <StyledIconClose
+            color="green300"
+            isCircled
+            onClick={this.handleBannerDismiss}
+          />
```
Comment (matejminar) [inline src/sentry/static/sentry/app/components/reprocessedBox.tsx:74]: This doesn't look very accessible, should we make it a button and/or add aria attributes?

File: src/sentry/static/sentry/app/views/organizationGroupDetails/groupDetails.tsx (hunk 1 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/views/organizationGroupDetails/groupDetails.tsx]: I think it's okay to leave it as it is, as the feature flag will be removed soon

**Review by matejminar — APPROVED** [review]

**Pull request merged**